internal/sourceanalysis/__snapshots__/integration_test.snap

```diff
@@ -124,7 +124,7 @@
           "position": {
             "filename": "\u003cAny value\u003e",
             "offset": -1,
-            "line": 840,
+            "line": 839,
```

Comment:
Do we want to set the line number to a fixed value? so we don't need to update it again when go version changes.

Reply:
I think it's alright for now, this allows us to check whether the line result in govulncheck is returning correctly. And we shouldn't be updating go that often anyway, so updating it when we do should be fine.